packages/@react-spectrum/dropzone/src/DropZone.tsx

```diff
@@ -41,13 +42,15 @@ function DropZone(props: SpectrumDropZoneProps, ref: DOMRef<HTMLDivElement>) {
   let {styleProps} = useStyleProps(props);
   let domRef = useDOMRef(ref);
   let messageId = useId();
+  let headingId = useId();
```

Comment:
Reid suggested to pass the `headingId` via slot which I would have liked to do but the problem is that it gets overridden in IllustratedMessage so it doesn't actually get passed all the way through

Reply:
I wonder why we decided to clear slots in Illustrated Message. Maybe we could test to see what happens when we remove that https://github.com/adobe/react-spectrum/pull/3420/files#diff-75401696e93859335f6081190bc047f0be8ff0b8e70f4639ae0e0944f8a3cf46R48-R52

Reply:
Yeah removing seems to work, though it looks like the reason for adding `<ClearSlots` was because we were concerned of possible collisions since IllustratedMessage uses the "illustration" slot and so does anything that's wrapped with `<Illustration>`

https://github.com/adobe/react-spectrum/pull/3420#discussion_r950492813